docker-compose.kafka.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.9'
+
+services:
+  kafka-01:
+    image: bitnami/kafka:3.4.0
+    hostname: "{{.Node.Hostname}}-{{.Service.Name}}"
+    environment:
+      BITNAMI_DEBUG: 'false'
+      KAFKA_ENABLE_KRAFT: 'yes'
+      KAFKA_KRAFT_CLUSTER_ID: 'ehB92ChxEe2iYQAAAkKsEg'
+      KAFKA_CFG_NODE_ID: 1
+      KAFKA_CFG_BROKER_ID: 1
+      KAFKA_CFG_PROCESS_ROLES: 'broker,controller'
+      KAFKA_CFG_CONTROLLER_LISTENER_NAMES: CONTROLLER
+      KAFKA_CFG_LISTENERS: PLAINTEXT://:9092,CONTROLLER://:9093
```

Comment:
Is this was working for you using port 9092 ? I was facing issues with this config and had to add additional configs